.dockerignore

```diff
@@ -0,0 +1,6 @@
+.venv
+*.pyc
+__pycache__/
+*.env
+.DS_Store
+Dockerfile
```

Comment:
Including `Dockerfile` in the `.dockerignore` file is unusual because it is needed to build the Docker image. Consider removing this line unless there is a specific reason to exclude the `Dockerfile` from the Docker context.